TaskManager/src/Kreta/TaskManager/Infrastructure/Symfony/Command/TaskFixturesCommand.php

```diff
@@ -40,7 +41,14 @@ public function __construct(CommandBus $commandBus, QueryBus $queryBus)
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        for ($i = 0; $i < 50; ++$i) {
+        $amount = 50;
+        $output->writeln('');
```

Comment:
I want to leave space above as its run after Project fixtures, same issue as before